src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -103,9 +103,16 @@ the new Certificate Authority [Let's Encrypt](https://letsencrypt.org/){: .exter
 
 ### User Gesture Required
 
-As a security feature, discovering nearby Bluetooth devices with
-`navigator.bluetooth.requestDevice` must be called via a user gesture
-like a touch or mouse click.
+As a security feature, discovering Bluetooth devices with
+`navigator.bluetooth.requestDevice` must be triggered by [a user
+gesture](https://html.spec.whatwg.org/multipage/interaction.html#activation)
+such as a touch or a mouse click. We're talking about listening to
+[`pointerup`](https://developers.google.com/web/updates/2016/10/pointer-events),
```

Comment:
Please remove the hard coded dev.google.com url and use a relative URL instead. Hard coded with full domain aren't allowed because it breaks the China version of Google Devs